packages/js/src/ui/components/elements/Bell/Bell.tsx

```diff
@@ -1,18 +1,19 @@
-import { Accessor, JSX } from 'solid-js';
+import { Show } from 'solid-js';
+import { BellMounter } from 'src/ui/types';
 import { useUnreadCount } from '../../../context/UnreadCountContext';
+import { ExternalElementMounter } from '../../ExternalElementMounter';
 import { BellContainer } from './DefaultBellContainer';
 
 type BellProps = {
-  //TODO: convert this in a `mountBell` prop like we do for notifications.
-  children?: ({ unreadCount }: { unreadCount: Accessor<number> }) => JSX.Element;
+  mountBell?: BellMounter;
 };
 /* This is also going to be exported as a separate component. Keep it pure. */
-export function Bell(props: BellProps) {
+export const Bell = (props: BellProps) => {
```

Comment:
Having it as a function breaks the TS for some reason.
it was not inferring the props type in the React package